docs/diagnostics/PX1062.md

```diff
@@ -0,0 +1,44 @@
+# PX1062
+This document describes the PX1062 diagnostic.
+
+## Summary
+
+| Code   | Short Description                                                                         | Type                           | Code Fix    | 
+| ------ | ----------------------------------------------------------------------------------------- | ------------------------------ | ----------- | 
+| PX1062 | You cannot declare static members in graphs or graph extensions                           | Error | Available | 
+
+## Diagnostic Description
+
+Static members in classes of the PXGraph type or its successors are strictly prohibited unless these members are read-only. The diagnostic shows the error for the following static members:
```

Comment:
I thought it would be more structured if I mention all types of fields. Also if somebody sees a more exact name such as "action", it may trigger more reponse than just a generic term such as "field".

Reply:
Yes, but the term "member" captures much more than just fields and properties. It is understood as any C# type member which includes constructors, events, methods and other more rare things which we allow to be static. We have tons of static methods in graphs and it is normal. 

Also the message mentions only graphs, we need to mention graph extensions too. The message is incorrect because we do not allow static actions and views even if they are read only. This works only for normal static fields and properties.

We can additionally specify actions and views separately in the message. This is why I suggested something like this:
```
In graphs and graph extensions static actions, views and mutable fields and properties are strictly forbidden.
```